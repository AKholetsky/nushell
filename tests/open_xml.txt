cd tests
open test.xml | get rss.channel.item.link | echo $it
exit
