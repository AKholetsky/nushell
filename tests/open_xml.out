http://www.jonathanturner.org/2015/10/off-to-new-adventures.html
